markets/bfp-market/contracts/modules/PerpAccountModule.sol

```diff
@@ -31,14 +31,16 @@ contract PerpAccountModule is IPerpAccountModule {
 
     struct Runtime_splitAccount {
         uint256 oraclePrice;
-        uint256 im;
+        uint256 toIm;
+        uint256 fromIm;
         uint128 debtToMove;
         int128 sizeToMove;
         uint256 supportedSynthMarketIdsLength;
         uint128 synthMarketId;
         uint256 collateralToMove;
         uint256 fromAccountCollateral;
         uint256 toCollateralUsd;
+        uint256 fromCollateralUsd;
```

Comment:
Addition of `fromIm` and `fromCollateralUsd` should have triggered a change in `storage.dump.sol`